backend/api/elite2Api.js

```diff
@@ -93,6 +93,11 @@ const elite2ApiFactory = client => {
   const getAlerts = (context, { agencyId, offenderNumbers }) =>
     post(context, `/api/bookings/offenderNo/${agencyId}/alerts`, offenderNumbers)
 
+  const getAlertsForBooking = (context, { bookingId, query }, headers) => {
+    contextProperties.setCustomRequestHeaders(context, headers)
+    return get(context, `/api/bookings/${bookingId}/alerts${query}`)
+  }
+
```

Comment:
I just noticed that we don't have any tests for elite2Api.js, but most of the other API clients have them. So nothing to do with your PR really, I just wondered why?

Reply:
The ones that have tests test logic which manipulates the response or the request in a way. Elite2 and Whereabouts purely pass on the response to the controller, whose tests mock it out.

Reply:
That isn't to say we couldn't add some tests, I'm sure there are things to test :D